nvm.sh

```diff
@@ -1820,12 +1820,14 @@ nvm_print_versions() {
   fi
 
   command awk \
-    -v remote_versions="$(printf '%s' "${1-}" | tr '\n' '|')" \
+    -v remote_versions="$(printf '%s' "${1-}" | tr '\n' '|')" -v min_ver="${NVM_MIN:-v0}" \
```

Comment:
is defaulting it to `v0` the right choice, as opposed to skipping the logic entirely?

Reply:
Surely we could do that, meanwhile versions like `v0`, `v0.0`, and `0.0.0` (with the prefix `v`), are all legit, therefore, we have the `filter_on` flag which is deduced with this:

    filter_on = (vcmp("v0.0.0", min_ver) != 0);

And given that the versions list is already sorted, once we encounter the version greater or equal to the min version specified, we turn off the filtering logic and show all versions onward.